src/transaction/log_manager.c

```diff
@@ -3348,6 +3349,38 @@ log_skip_logging_set_lsa (THREAD_ENTRY * thread_p, LOG_DATA_ADDR * addr)
   return;
 }
 
+/* log_pack_log_boot_info - pack together the log header, log append pages
+ *              and prev lsa for the purpose of sending them to the passive transaction server
+ *              as part of the log initialization sequence
+ */
+// *INDENT-OFF*
+std::string log_pack_log_boot_info (THREAD_ENTRY * thread_p)
```

Comment:
I would move this function to async_page_fetcher. `log_manager` is for generic log management functions. `log_pack_log_boot_info` is specifically used only on the page server, to serve a the booting of a new passive transaction server (possibly active transaction servers too in the future).

Reply:
It could be done. But it leads to a lot of link errors in unit tests. Another options is to move this to a separate `log_....c` file.

Reply:
Let's leave it as is for now.